rust/src/types/commitment.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2025 Sonic Operations Ltd
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at soniclabs.com/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+/// A Verkle trie node commitment.
+pub trait Commitment {}
+
+/// In case we don't store an commitment, we use the unit type.
```

Comment:
```suggestion
/// In case we don't store a commitment, we use the unit type.
```

Is this a use case we discussed? When would you want to not store a commitment?

Reply:
As discusses, we don't need the trait as we will be using always the same commitment type.